src/assets/sass/scratch/_toolbox.scss

```diff
@@ -34,82 +32,118 @@ $category-colours: (
     overflow: hidden;
     position: absolute;
     user-select: none;
-    z-index: 99 !important;
+    z-index: 1;
+    width: 250px;
 
     &__header {
+        padding: 0.5em;
+        position: relative;
         font-weight: bold;
+        font-size: 1.6em;
         background-color: $grey;
+        text-align: center;
+    }
+    &__item {
+        display: flex;
+        flex-direction: row;
+    }
+    &__cat-arrow {
+        display: flex;
+        margin: auto;
+
+        &--back {
+            margin-left: $cat-text-spacing;
+            fill: $white;
+            transform: rotate(90deg);
```

Comment:
@aaron-binary can you please use a mixin for `transform` and browser support.

Reply:
I've added the mixin as part of https://github.com/binary-com/deriv-bot/pull/14